src/navigation/app/app-navigation.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import { AppRoute } from '../../common/enums/navigation';
+import HomeScreen from '../../screens/Home';
+import ExploreScreen from '../../screens/Explore/ExploreScreen';
+import OpportunitiesScreen from '../../screens/Opportunities/OpportunitiesScreen';
+import OKRScreen from '../../screens/OKR/OKRScreen';
+import ProfileScreen from '../../screens/Profile/ProfileScreen';
+import styles from './styles';
+
+const Tab = createMaterialBottomTabNavigator();
+
+const AppNavigation = () => {
+  return (
+    <Tab.Navigator
+      shifting={false}
+      initialRouteName={AppRoute.HOME}
+      barStyle={styles.barStyle}
+      inactiveColor="#796EFC"
+    >
+      <Tab.Screen
+        name={AppRoute.HOME}
+        component={HomeScreen}
+        options={{
+          tabBarIcon: ({ color }) => (
+            <MaterialCommunityIcons name="home" color={color} size={25} />
```

Comment:
Let's take all icons to /src/components/icons.tsx
So that you can just import it here as <CompassIcon /> and use.